test/runtests.jl

```diff
@@ -1315,6 +1339,27 @@ end
                 result = execute(conn, "SELECT '(2018-01-01, 2018-02-02]' = \$1", [Interval(Date(2018, 1, 1), Date(2018, 2, 2), false, true)])
                 @test first(first(result))
                 close(result)
+
+            end
+
+            @testset "InfExtendedTime" begin
+                tests = (
+                    ("'infinity'::date", InfExtendedTime{Date}(∞)),
+                    ("'-infinity'::date", InfExtendedTime{Date}(-∞)),
+                    ("'infinity'::timestamp", InfExtendedTime{DateTime}(∞)),
+                    ("'-infinity'::timestamp", InfExtendedTime{DateTime}(-∞)),
+                    ("'infinity'::timestamptz", InfExtendedTime{ZonedDateTime}(∞)),
+                    ("'-infinity'::timestamptz", InfExtendedTime{ZonedDateTime}(-∞)),
+                    ("'(-infinity, 2012-01-01]'::daterange", Interval{Open, Closed}(-∞, Date(2012, 1, 1))),
+                    ("'(2012-01-01, infinity]'::daterange", Interval{Open, Closed}(Date(2012, 1, 1), ∞)),
+                    ("'(-infinity, infinity)'::tstzrange", Interval{Open, Open}(InfExtendedTime{ZonedDateTime}(-∞), InfExtendedTime{ZonedDateTime}(∞)))
+                   )
+
+                for (pq_str, obj) in tests
```

Comment:
```suggestion
                @testset for (pq_str, obj) in tests
```
Otherwise we'll get a test failure but we won't know for which test